crates/extension/src/manifest/v1.rs

```diff
@@ -62,6 +85,47 @@ mod tests {
     use super::*;
     use serde_json::json;
 
+    #[test]
+    fn permissions() {
+        let json = json!({
+            "id": {"name": "test", "version": "1.0.0"},
+            "kind": {
+                "FieldResolver": {
+                    "resolver_directives": ["custom"]
+                }
+            },
+            "sdl": "directive @custom on FIELD_DEFINITION",
+            "sdk_version": "0.1.0",
+            "minimum_gateway_version": "0.1.0",
+            "description": "Mandatory description",
+            "homepage_url": "http://example.com/my-extension",
+            "permissions": ["stdout", "stderr", "environment_variables", "network"]
```

Comment:
I'm wondering, will we need parameters inside some of these in the future, like "only these env vars", or "only this domain, network wise"

Reply:
We can worry about that in manifest v2 maybe

Reply:
Maybe, we don't know yet.